extensions/mongodb/src/main/java/com/hazelcast/jet/sql/impl/connector/mongodb/BsonTypes.java

```diff
@@ -156,7 +157,7 @@ private static Map<Class<?>, BsonType> generateJavaClassToBsonTypeMapping() {
         return result;
     }
 
-    @SuppressWarnings("checkstyle:ReturnCount")
+    @SuppressWarnings({"checkstyle:ReturnCount", "checkstyle:RightCurly"})
```

Comment:
What's this?

Reply:
when you use if, you have to use 
```
if () {
something();
} else if (other) {
other();
}
```
Right Curly means you can do:
```
if () {
something();
} 
else if (other){
other();
}
```
IMHO it's clearer in long chains of if-else